etc/docker/Dockerfile

```diff
@@ -39,8 +39,17 @@ WORKDIR /app
 RUN apt-get update
 RUN apt-get install -y git lsb-core sudo
 
+RUN git reset --hard && git clean -xfd
+RUN rm -rf /app/._build/
```

Comment:
will this be cleaned by `git clean -dfx`?

Reply:
No, it has to be manually removed.